backend/src/tasks/dnstwist.ts

```diff
@@ -30,13 +28,45 @@ async function runDNSTwist(domain: Domain) {
 export const handler = async (commandOptions: CommandOptions) => {
   const { organizationId, organizationName } = commandOptions;
 
+  await connectToDatabase;
+  const date = new Date(Date.now());
   console.log('Running dnstwist on organization', organizationName);
   const domainsWithIPs = await getIps(organizationId);
 
   const vulns: Vulnerability[] = [];
   for (const domain of domainsWithIPs) {
     try {
       const results = await runDNSTwist(domain);
+      const existingVulns = await Vulnerability.find({
```

Comment:
Since we're assuming that there's only one existing vuln (max), let's just use Vulnerability.findOne instead.